specification/cognitiveservices/OpenAI.Inference/routes.tsp

```diff
@@ -1,21 +1,24 @@
 import "@azure-tools/typespec-azure-core";
 import "@typespec/rest";
 import "@typespec/http";
-//import "@azure-tools/typespec-client-generator-core";
+import "@typespec/versioning";
 
 import "./models/completions.create.tsp";
 import "./models/chat.completions.tsp";
 import "./models/embeddings.create.tsp";
+import "./models/images.tsp";
 
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
+
 using Azure.Core;
-//using Azure.ClientGenerator.Core;
 
 namespace Azure.OpenAI;
 
 @doc("Return the embeddings for a given prompt.")
-@TypeSpec.Rest.actionSeparator("/")
+@added(ServiceApiVersions.v2022_12_01)
```

Comment:
This `@added` is not needed - every type without a versioning decorator is included in the first api-version